nixos/modules/services/mail/rss2email.nix

```diff
@@ -0,0 +1,214 @@
+
+          This set contains key-value associations that will be set in the
+          <literal>[DEFAULT]</literal> block along with the
+          <literal>to</literal> parameter.
+        '';
+        example = lib.literalExample ''
+          # This example is the default configuration as of rss2email-3.9
+          # If you set nothing, default values will be used
+          {
+            use-8bit = false;
+            force-from = false;
+            use-publisher-email = false;
+            name-format = "{feed-title}: {author}";
+            proxy = "";
+            feed-timeout = 60;
+            active = true;
+            digest = false;
+            date-header = false;
+            date-header-order = "modified, issued, created, expired";
+            bonus-header = "";
+            trust-guid = true;
+            trust-link = false;
+            encodings = "US-ASCII, ISO-8859-1, UTF-8, BIG5, ISO-2022-JP";
+            post-process = "";
+            digest-post-process = "";
+            html-mail = false;
+            use-css = false;
+            css = builtins.replaceStrings ["\n"] [""] '''
+              h1 {
+                font: 18pt Georgia, "Times New Roman";
+              }
+              body {
+                font: 12pt Arial;
+              }
+              a:link {
+                font: 12pt Arial;
+                font-weight: bold;
+                color: #0000cc;
+              }
+              blockquote {
+                font-family: monospace;
+              }
+              .header {
+                background: #e0ecff;
+                border-bottom: solid 4px #c3d9ff;
+                padding: 5px;
+                margin-top: 0px;
+                color: red;
+              }
+              .header a {
+                font-size: 20px;
+                text-decoration: none;
+              }
+              .footer {
+                background: #c3d9ff;
+                border-top: solid 4px #c3d9ff;
+                padding: 5px;
+                margin-bottom: 0px;
+              }
+              #entry {
+                border: solid 4px #c3d9ff;
+              }
+              #body {
+                margin-left: 5px;
+                margin-right: 5px;
+              }
+            ''';
+            unicode-snob = false;
+            links-after-each-paragraph = false;
+            body-width = 0;
+            email-protocol = "sendmail";
+            sendmail = "sendmail";
+            smtp-auth = false;
+            smtp-username = "username";
+            smtp-password = "password";
+            smtp-server = "smtp.yourisp.net:25";
+            smtp-ssl = false;
+            smtp-ssl-protocol = "SSLv3";
+            imap-auth = false;
+            imap-username = "username";
+            imap-password = "password";
+            imap-server = "imap.yourisp.net";
+            imap-port = 143;
+            imap-ssl = false;
+            imap-mailbox = "INBOX";
+            verbose = "warning";
```

Comment:
Now that we don't encode the default in the module anymore, I think this here should be removed as well and potentially link to some upstream default docs if available.

Reply:
Indeed, and actually these are in the manpage… or should be, if rss2email had had a release in the last 3 years. Well, with @Profpatsch we now have access to rss2email, so should be able to cleanup and make a release as soon as we get some time :)

I've pointed to the online `r2e.1` for the time being.